labels.json

```
@@ -87,6 +87,14 @@
     "name": "🧹 status: ticket work required",
     "color": "666666"
   },
+  {
+    "name": "🙅 status: discontinued",
+    "color": "666666"
+  },
+  {
+    "name": "🏷 status: label work required",
+    "color": "666666"
```

Here I suggest using this color `#F4EE00`.
```suggestion
    "color": "#F4EE00"
```

---

Sure!